src/core/public/chrome/chrome_service.tsx

```diff
@@ -158,6 +161,29 @@ export class ChromeService {
         }
         this.collapsibleNavHeaderRender = render;
       },
+      registerNavLink: (useCase: ChromeUseCase, navLink: ChromeRegistrationNavLink) => {
+        const currentUseCaseList = [...this.useCaseList$.getValue()];
+        const findItem = currentUseCaseList.find((item) => item.id === useCase.id);
```

Comment:
can we use a meaningful name instead of`findItem`?

Reply:
Sure, changed the code here.